core/revapi.json

```diff
@@ -5539,6 +5539,1284 @@
         "old": "missing-class org.apache.tinkerpop.gremlin.structure.VertexProperty",
         "new": "missing-class org.apache.tinkerpop.gremlin.structure.VertexProperty",
         "justification": "JAVA-2907: switched Tinkerpop dependency to optional"
+      },
```

Comment:
API changes brought on by the Jackson upgrade... apparently some of the class hierarchy was re-worked, especially around the exceptions.